docsrc/Makefile

```diff
@@ -22,7 +22,6 @@ github:
 	@make html
 	@rm -rf ../docs/*
 	@cp -r _build/html/* ../docs
-	@rm -rf ../docs/_modules
 	@rm -rf ../docs/_sources
```

Comment:
```suggestion

```

Sorry, by "leave it", I think we should remove this command and leave the _sources folder